cluster/src/main/java/com/hartwig/pipeline/resource/RefGenome37ResourceFiles.java

```diff
@@ -156,6 +156,9 @@ public String genotypeSnpsDB() {
         return formPath(GENOTYPE_SNPS, "26SNPtaq.vcf");
     }
 
+    @Override
+    public String genotypeMipSnpsDB() { return formPath(GENOTYPE_SNPS, "31SNPtaq.vcf"); }
```

Comment:
This is on a single line. All other methods are formatted spread across three lines.